app/packages/core/src/crds/dashboard.ts

```diff
@@ -1,6 +1,8 @@
+import { TTimeQuick } from '../utils/times';
+
 export interface IDashboard {
   cluster: string;
-  defaultTime?: string;
+  defaultTime?: TTimeQuick;
```

Comment:
I think this can lead to problems, since this is a user provided value from the CRs we can not be sure that the string is really of the type we want.

Reply:
Want me to validate the string and throw an error? Before my change we just did a type cast which is worse:
https://github.com/kobsio/kobs/pull/477/files#diff-c798aa39ca7436f39384b3767b84943594af5380e847070f7f05b2abaa26f4c0R214-R215